client/configuration/eslint/eslint.js

```diff
@@ -101,6 +101,7 @@ module.exports = {
       "error",
       { ignoreVoid: true, ignoreIIFE: true },
     ],
+    "jsx-a11y/label-has-associated-control": ["warn"],
```

Comment:
This is to avoid the eslint error `A form label must be associated with a control.` -> [here](https://github.com/chanzuckerberg/single-cell-explorer/blob/main/client/src/components/categorical/value/index.tsx#L604)

Reply:
Ah I think we can fix the a11y issue!

https://github.com/chanzuckerberg/single-cell-explorer/pull/1087

Reply:
Thanks again @tihuan !